### DISCUSSION TITLE
Pull Requests Review Requested hangs

### DISCUSSION THREAD

**Discussion by mih-kopylov** [community/community#44070] [Mobile] [answered]
Pull Requests Review Requested opens first time, but when opening 9ne of them and going back to the list, it never loads again.
Just endlessly rolling circle progress is shown.

![Screenshot_20230112_082136_GitHub.jpg](https://user-images.githubusercontent.com/6898003/211984367-fc38fd7d-e645-4702-8ecb-e985eedeec96.jpg)

**Comment by alcere** [comment, marked as answer]
@mih-kopylov thanks for the report! I have shared the issue with the team and they will take a look to it.
